colossalai/shardformer/modeling/jit.py

```diff
@@ -0,0 +1,34 @@
+import torch
+
+
+def get_dropout_add_func():
+
+    from transformers.models.bloom.modeling_bloom import dropout_add
+
+    def self_dropout_add(self, x: torch.Tensor, residual: torch.Tensor, prob: float, training: bool) -> torch.Tensor:
+        return dropout_add(x, residual, prob, training)
+
+    return self_dropout_add
+
+
+def get_jit_fused_dropout_add_func():
+
+    from colossalai.kernel.jit import bias_dropout_add_fused_inference, bias_dropout_add_fused_train
```

Comment:
Does JIT ops have any compatibility problem?

Reply:
> Does JIT ops have any compatibility problem?

I haven't encountered any compatibility problem with the JIT operators so far.

Reply:
> Does JIT ops have any compatibility problem?

Updated